stable_baselines3/common/policies.py

```diff
@@ -617,6 +617,63 @@ def __init__(self,
                                                    optimizer_kwargs)
 
 
+class ContinuousCritic(BasePolicy):
+    """
+    Critic network for DDPG/SAC/TD3.
+    It represents the action-state value function (Q-value function).
+    By default, it creates two critic networks to reduce overestimation
+    using clipped Q-learning (cf TD3 paper).
```

Comment:
Doc here makes me think this critic does the overestimation-fix. Perhaps it should say "two critic used to reduce using clipped Q-learning"?